### DISCUSSION TITLE
Windows, please

### DISCUSSION THREAD

**Discussion by ErSpieltHD** [isontheline/pro.webssh.net#744] [Q&A] [answered]
Please bring this to windows' PCs.

It doesn't have to be much. Just a little shell/cmd/dos window and the features from the app. Like ssh private keys, and the snippets.

I really love the snippets feature.

**Comment by isontheline** [comment]
@ErSpieltHD Thank you so much for your support about WebSSH 🙏

On windows you can use the awesome [MobaXterm](https://mobaxterm.mobatek.net)

**Comment by ErSpieltHD** [comment, marked as answer]
Hm. I don't like the design of it.

I think I'll keep using your (great) app or PuTTY on PC.

**Reply by isontheline** [reply]
Thank you a lot @ErSpieltHD
